fpga_interchange/fasm_generators/xc7.py

```diff
@@ -15,6 +15,16 @@
 from fpga_interchange.fasm_generators.generic import FasmGenerator
 from fpga_interchange.route_stitching import flatten_segments
 from fpga_interchange.physical_netlist import PhysicalPip
+"""
```

Comment:
There should probably be a top level block comment before the one for ExtraFeatures.

Reply:
Yep, makes sense, added general file comment